src/main/java/com/toofifty/easyblastfurnace/utils/BarsOres.java

```diff
@@ -53,8 +36,12 @@
         this.itemID = itemID;
     }
 
-    public static BarsOres getVarbit(int varbit)
```

Comment:
Hmm is this not still needed?

Reply:
`setFurnaceCount(BarsOres.GOLD_ORE.getVarbit(), 1);` Lines like this work just fine thanks to the `@Getter`

```Java
    @Getter
    private final int varbit;
```